README.md

```diff
@@ -2,54 +2,69 @@ EOS - A HEP Program for Flavor Observables
 ==========================================
 
 EOS is a software package that addresses several use cases in the field of
-high-energy flavor physics:
+high-energy flavor physics (HEP):
 
- - calculation and uncertainty estimation of flavor observables within
-   various models, and
- - Bayesian inference of parameters from experimental and/or theoretical
-   constraints.
+ 1. calculation and uncertainty estimation of flavor observables within
+   various models,
+ 2. Bayesian inference of parameters from experimental and/or theoretical
+   constraints, and
+ 3. sampling process-specific probability density functions.
 
-EOS is written in C++, and relies on as a small set of dependencies:
+An up-to-date list of EOS related publications can be found [here](https://eos.github.io/publications/).
 
- - GNU Scientific Library
- - Hierarchical Data Format v5 (HDF5)
- - Minuit2 v5 (or greater)
- - libPMC (optional)
+EOS is written in C++14, with an optional interface to Python, and depends on
```

Comment:
You write C++14 but in `configure.ac` we still check for C++0x. Then it seems it's not C++14. Can you please check?

Reply:
```configure.ac``` has already been changed to check for and use ```-std=c++14```.